Adding tile example to Marketplace docs (#13997)
DataDog/documentation: content/en/developers/marketplace/_index.md

3. SaaS license or subscription (Marketplace only)
4. Professional services (Marketplace only)

> Regardless of the type of offering that you are developing, you will build out a tile on the Marktplace or Integrations page that represents your listing. You can see an example of a Marketplace tile below:

Reviewer: ```suggestion
Create an image to use for the tile on the Marketplace or Integrations page that represents your listing. You can see an example of a Marketplace tile below:
```

Reviewer: Per discussion in Slack, you can keep the original language, but please remove the "will" so the sentence is in present tense.

Author: Thanks @urseberry , updated in latest commit
